source/index.html

```diff
@@ -32,39 +34,41 @@
     </nav>
   </header>
   <main class="page-main">
-    <img src="http://via.placeholder.com/320x386" alt="Добро пожаловать в великолепную Седону. Потому что Гранд Каньон - отстой!">
-    <h1 class="page-main__header">СЕДОНА — небольшой городок в Аризоне, заслуживающий большего!</h1>
+    <div class="page-main__slogan">
+      <img src="http://via.placeholder.com/282x214" alt="Добро пожаловать в великолепную Седону. Потому что Гранд Каньон - отстой!">
+    </div>
+    <h1 class="page-main__header">Седона — небольшой городок в Аризоне, заслуживающий большего!</h1>
     <section class="features">
       <div class="features__wrapper">
         <h2 class="visually-hidden">Преимущества Седоны</h2>
         <p class="features__intro">Рассмотрим 5 причин, по которым Седона круче, чем Гранд Каньон!</p>
         <ul class="features__list">
           <li class="features__item">
-            <div class="item-block">
+            <div class="features__item-block">
               <div class="features__card features__card--main">
                 <h3 class="features__title">Настоящий городок</h3>
                 <p class="features__number">— №1 —</p>
                 <p class="features__description">Седона не аттракцион для туристов, там течет своя жизнь</p>
               </div>
               <img src="http://via.placeholder.com/320x260" class="features__image features__image--city" alt="Улица Седоны. Зелёное двухэтажное здание с магазинами">
             </div>
-            <ul class="recommendation-list">
-              <li class="recommendation-list__item recommendation-list__item--housing">
-                <h4 class="recommendation-list__header">Жилье</h4>
-                <p class="recommendation-list__description">Рекомендуем пожить в настоящем мотеле, все как в кино!</p>
+            <ul class="features__recommendation-list">
+              <li class="features__recommendation-item features__recommendation-item--housing">
+                <h4 class="features__recommendation-header">Жилье</h4>
+                <p class="features__recommendation-description">Рекомендуем пожить в настоящем мотеле, все как в кино!</p>
               </li>
```

Comment:
Давай подсократим названия чтобы стало читаемее
```suggestion
            <ul class="features__list">
              <li class="feature feature--housing">
                <h4 class="features__header">Жилье</h4>
                <p class="features__description">Рекомендуем пожить в настоящем мотеле, все как в кино!</p>
              </li>
```

Reply:
Вообще при нейминге старайся использовать короткие слова и конструкции. Длинные слова в будущем скажутся на читаемости:
```javascript
class="very-long-block__very-long-element very-long-block__very-long-element--very-long-modificator"
```
 Это утрированный пример, но все же подобные случаи встречаются. Все четко и по БЭМу, а читать сложно. Вдобавок объем кода растет

Если видишь, что получилось длинно, старайся сократить или перефразировать, как правило это всегда возможно

Reply:
далее по коду есть классовая конструкция `features__item features__card features__card--sub`. 
Уверен, что нам пригодятся доп блок `features__card`? Может выйдет ограничиться модификатором от `features__item`?
Пардон, упустил это в прошлый раз

Reply:
Т.к это список внутри списка, то features__list и т.д уже занято, поэтому для сокращения изменю recommendation на advice

Reply:
Согласен, можно оставить только модификатор для features___item. Но всё равно придётся оставить дополнительный блок-обёртку, чтобы назначит синий цвет фона для двух преимуществ, отделив его от изображения.

Reply:
Подумал, всё-таки пока что не понимаю, как заменить features__card из-за наличия картинок в двух элементах. А стили нужны именно для блока с названием, номером и описанием причины. Даже если дать фон всему блоку и перекрыть его картинкой, то с внутренними отступами так не поступить. Возможно пойму, когда буду полностью стилизовать